integrations/serverpod_cloud_storage_gcp/pubspec.yaml

```diff
@@ -24,5 +24,4 @@ dependencies:
   xml2json: ^5.3.6
 
 dev_dependencies:
-  serverpod_lints: #1.1.0
-    path: ../../packages/serverpod_lints
+  lints: ^2.0.1
```

Comment:
Question: What is the reason for not using the Serverpod lint package for these packages as well?